## this file is similar to octomap_pa/srv/OctomapPaFileName.srv

string filename
---
bool ok
